src/components/application_manager/test/commands/mobile/register_app_interface_request_test.cc

```diff
@@ -201,7 +205,41 @@ class RegisterAppInterfaceRequestTest
   MockHmiInterfaces mock_hmi_interfaces_;
 };
 
-TEST_F(RegisterAppInterfaceRequestTest, DISABLED_Run_MinimalData_SUCCESS) {
+MATCHER_P2(CheckLockScreenIconUrlNotification, connection_key, url, "") {
+  const bool is_request_type_correct =
+      mobile_apis::RequestType::LOCK_SCREEN_ICON_URL ==
+      (*arg)[am::strings::msg_params][am::strings::request_type].asInt();
+  const bool is_function_id_correct =
+      mobile_apis::FunctionID::OnSystemRequestID ==
+      (*arg)[am::strings::params][am::strings::function_id].asInt();
```

Comment:
@LevchenkoS please cache the `(*arg)[am::string::params]`
code is kind  of unredable now